project-template/index.template.js

```diff
@@ -1 +1,17 @@
-console.log('I am the bot.');
+const { registerModule } = require('nay');
+
+function onReceive(sender) {
+  // TODO
+}
+
+function onEnable() {
+  // TODO
+}
+
```

Comment:
Another thought: maybe we should add two templates or make one, but simpler template.
E.g. we can make a simple module with just one `onEnable(sender)` and a rich module with all three methods.

What do you think? Or we can leave it for now and resolve later.

Reply:
I think user should be able to omit handlers. If some handlers are not defined, we can skip them.

Reply:
I definitely agree with you. The documentation on additional functions will be enough, don't need to have them empty.